src/utils/worker/WorkerPool.ts

```
@@ -0,0 +1,142 @@
+    this.initializePool();
+  }
+
+  private initializePool() {
+    for (let i = 0; i < this.poolSize; i++) {
+      try {
+        const worker = this.workerFactory();
+        worker.addEventListener('message', this.handleWorkerMessage.bind(this));
+        worker.addEventListener('error', this.handleWorkerError.bind(this));
+        // Add message error handler
+        worker.addEventListener('messageerror', (event) => {
+          console.error('Worker message error:', event);
```

ESLint

[error] Unexpected console statement.

(no-console)